tests/acceptance/new.spec.js

```diff
@@ -81,16 +81,6 @@ describe('Acceptance: ng new', function () {
     });
   });
 
-  it('Cannot run ng new, inside of ember-cli project', function () {
```

Comment:
Don't really know the reason but seems like `ember-cli` now allows that kind of things. Let me check if I shoud `catch` here too...